stack/stack_noaa.yaml

```diff
@@ -47,7 +47,7 @@ pnetcdf:
 
 netcdf:
   build: YES
-  shared: NO
+  shared: YES
```

Comment:
please revert.

Reply:
See comment above.

Reply:
Reverted to static-only in f8afa5b128a5b3e85239b652999891fe56c7e1b4.